averbis/core/_rest_client.py

```diff
@@ -632,6 +735,7 @@ def get_type_system(self) -> TypeSystem:
         return self.cached_type_system
 
     @experimental_api
+    @deprecated("Use resource container API instead.")
```

Comment:
Do we see the deprecation warning in generated documentation?
If not, the deprecation notice should probably also go into the docstring.
I think the pointer to which function specifically to use as a replacement could be clearer. Can we include a link to the new method that can be properly rendered as a link in the generated documentation?